examples/getting-started/Vagrantfile

```diff
@@ -3,6 +3,11 @@
 
 Vagrant.require_version ">= 2.0.0"
 
+# Note: Cilium version must remain pinned to a v1.4 branch version.
```

Comment:
:( this is suboptimal. Is there no way for us to install the binary via `curl` in these VMs in the meantime?

Reply:
It's possible, but for getting started guide all we need is a version of Cilium to spin up with all the standard features. The minimum we can do is document so that we at least shouldn't break the GSG VM without noticing.